lib/harvest_utils.rb

```diff
@@ -2,6 +2,8 @@
 require "open-uri"
 require "fileutils"
 require "oai"
+require "awesome_print"
```

Comment:
@skng5 You left a reference to `awesome_print` in here, but it isn't in the Gemfile.